### DISCUSSION TITLE
https://zhangzhao219.github.io/2023/02/19/diary/Happy-Birthday-2023/

### DISCUSSION THREAD

**Discussion by giscus** [zhangzhao219/zhangzhao219.github.io#9] [Announcements]
# https://zhangzhao219.github.io/2023/02/19/diary/Happy-Birthday-2023/

23岁的自己，生日快乐！

https://zhangzhao219.github.io/2023/02/19/diary/Happy-Birthday-2023/

**Comment by kohler-19** [comment]
生日快乐！学长，在学习您当时的模式识别文章，学长写得很棒。   
![](https://s21.ax1x.com/2024/09/19/pAK4O0I.png)
